apps/dashboard/.env.example

```diff
@@ -1,7 +1,7 @@
 # Supabase setup - Required
 PUBLIC_SUPABASE_PROJECT_REF=localhost
-PUBLIC_SUPABASE_URL=http://localhost:54321
-PUBLIC_SUPABASE_ANON_KEY=
```

Comment:
Please can you undo the changes in this file?

Use the content here

https://github.com/rotimi-best/classroomio/blob/main/apps/dashboard/.env.example

Reply:
Yes , i already did